velox/dwio/dwrf/writer/Writer.cpp

```diff
@@ -35,7 +37,52 @@ namespace facebook::velox::dwrf {
 
 void WriterOptions::processConfigs(
     const config::ConfigBase& connectorConfig,
```

Comment:
How about we pass connectorConfig through a shared ptr to avoid a copy and the same for session config? Thanks!